package.json

```diff
@@ -18,12 +20,17 @@
     "react-dom": "^15.4.2"
   },
   "devDependencies": {
+    "@types/node": "^7.0.5",
     "@types/react": "^15.0.0",
     "@types/react-dom": "^0.14.20",
-    "@types/node": "^7.0.5",
-    "@types/whatwg-fetch": "0.0.33",
     "awesome-typescript-loader": "^3.0.0-beta.18",
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
```

Comment:
All these `babel` just to compile into ES5? Can `webpack` do it? I remember seeing some settings in `webpack.config` which tells what's the JavaScript version for output.